tests/test_csrf_extension.py

```diff
@@ -30,6 +32,24 @@ def csrf(app):
     return app.extensions["csrf"]
 
 
+def test_validate_csrf(app):
+    with app.test_request_context():
+        token = generate_csrf()
+        assert validate_csrf(token) is None
+
+
+def test_validate_csrf_legacy_flask_wtf_013(app):
+    # Test confirms we can validate csrf tokens generated by flask-wtf 0.13
+    with app.test_request_context():
+        session['csrf_token'] = "static csrf token"
+        legacy_token1 = "2147400000##8587b4e882f4f9ca8dbe764657a839b10b6ce782"
+        assert validate_csrf(legacy_token1, secret_key='dev') is None
+
+        legacy_token2 = "##12a714b52cf57340c08dcab228f89c453399a2b4"
+        assert validate_csrf(legacy_token2, time_limit=0, secret_key='dev') is None
```

Comment:
`time_limit=0` here because the stuff before `##` is the expiration, and in this case we have no expiration to compare against.

see https://github.com/benchling/aurelia/blob/dev/tests/unit/flask_test.py#L140